source/Octopus.Tentacle.Tests.Integration/Support/TentacleTypesAndCommonVersionsToTest.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Collections;
+using Octopus.Tentacle.Contracts;
```

Comment:
What happened here to require the Contracts to be referenced?

Reply:
I dunno tbh, I think Rider may have auto-added it when I made a typo (I made some changes here, then reverted). Rider also seems to auto-hide the `using` statements on my machine, so I didn't notice it.

I've updated my Rider settings for next time, and I'll remove this.